app/pb_kits/playbook/pb_date_stacked/docs/_date_stacked_not_current_year.jsx

```diff
@@ -13,7 +13,7 @@ const DateStackedNotCurrentYear = () => {
       <br />
 
       <DateStacked
-          date={new Date('20 Mar 2018')}
+          date="2018-03-20"
```

Comment:
We still need to show this works off a date object. Can you either add it back or add another example using a string?

Reply:
That's here already: https://github.com/powerhome/playbook/pull/606/files#diff-b8e86b8e3fe5e0700f457960999baf69R9

(line 9 on the same file)

Reply:
Ahh sorry, I missed that. Perfect then!

Reply:
Thanks guys!